Documentation/EngineeringPlanLinuxDistroRepos.md

```diff
@@ -0,0 +1,132 @@
+>    * _Fedora_
+>    * _Ubuntu_
+>    * _CentOS_
+>    * _Debian_
+>* _Update source-build to create rpm|deb assets for submission into distro archive acceptance process_
+>* _Work through the first distro submission process_
+>* _Add distro archive maintenance to the .NET Core servicing and release processes_
+
+One of the 'Golden' distros, Fedora, has an existing special interest group (SIG) that maintains spec files for building .NET Core RPMs.  Their goal is to make these the official .NET Core packages in Fedora, but the source does not meet the Fedora packaging guidelines.  We should target Fedora as the first 'Golden' distro so we can focus on getting the source to a state that meets the packaging guidelines while the DotNet SIG can continue to focus on packaging for Fedora.  https://fedoraproject.org/wiki/SIGs/DotNet
+
+
+## Work Breakdown ##
+With this partnership, there are four main area of focus to get .NET Core ready for inclusion into Fedora's package archive:
+1. Prove out and define process for toolset bootstrapping.(https://github.com/dotnet/source-build/issues/724)
+2. Split runtime build and SDK build and define dependencies correctly. (https://github.com/dotnet/source-build/issues/725)
+3. Eliminate prebuilt binaries. (https://github.com/dotnet/source-build/issues/753)
+4. Work with DotNet SIG to get packaging ready.
+
+As stated in the requirements, work will be done in the `release/2.1` branch of source-build.  Any changes required to source repos will follow this process:
+1. Include changes in source-build as patches to continue moving work forward.
+2. Write up issue in target repo to include change in `release/2.1` branch and port to `release/2.2` and `master` branches.
+3. Update source-build to move to updated code and remove patch.
+
+Source-build changes will be ported to `release/2.2` and `master` on a regular basis to keep those branches up to date.  The desire is to have `release/2.2` and `master` updated and ready to be included in other distros when they are ready to be released.
+
+### Toolset Bootstrapping ##
+This work requires setting up scripts to build the .NET Core toolset from source and then using the result to build source-build.  This is to define the process and to ensure that it works consistently by adding CI.  It also will require working out issues that we've encountered building source-build with the same version that is being built.  See https://github.com/dotnet/source-build/issues/606.  This will also track work with source-building buildTools and the eventual move to Arcade, if it is ported to the `release/2.1` branch for any repos.
+
+### Runtime / SDK Split ###
+The main part of this work is to split and isolate the runtime and sdk builds into separate directories with separate build scripts in source-build.  The result of the runtime build should be input to the sdk build.  Each should be able to be built independently.
```

Comment:
I'm not sure separate directories is the best way to handle this, but I think this is just an implementation detail. Maybe leave that specific bit out from this doc and link to an issue here?